ATTRIBUTION.md

```diff
@@ -17,6 +17,120 @@ This package contains a variety of materials, including data sets and related ma
 * Location: ***prereise/gather/data/ba_to_county.txt***
 
 
+#### US County Map Shape Files
+#### Source
+* Name: US County map
+* Author: US Census Bureau
+* Description: US county map
+* Source: https://www.census.gov/
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): None
+* Location: ***prereise/gather/data/cb_2020_us_county_500k.zip***
+
+
+#### Retail Service Territory 
+#### Source
+* Name: Electric Retail Service Territory 
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: electric retail service territory
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-retail-service-territories-2/explore
+#### Destination
+* Modifications to source files(s): none
+* Location: ***prereise/gather/data/remap_ba_area/cb_2020_us_county_500k.zip***
+
+
+#### Upstate Subregions of NYISO
+#### Source
+* Name: upstate subregions of NYISO
+* Author: ArcGIS
+* Description: upstate subregions of NYISO
+* Source: https://www.arcgis.com/
+* Exact source location: https://www.arcgis.com/home/item.html?id=6fd1de467b134f47a607721f23a69f0c
+#### Destination
+* Modifications to source files(s): None for the upstate subregions A-F, they are directly downloaded by the program 
+* Location: ***prereise/gather/data/remap_ba_area/nyiso/***
+
+#### Downstate Subregions of NYISO
+#### Source
+* Name: downstate subregions of NYISO
+* Author: US Census Bureau
+* Description: downstate subregions of NYISO
+* Source: https://www.census.gov/
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): Downstate subregions G_I are made by grouping and splitting county shapes.
+* Location: ***prereise/gather/data/remap_ba_area/nyiso/NYISO_downstate.geojson***
```

Comment:
If the shapes are created solely via county shapes, do we need another input file, or can we include the changes in the existing `BA_county_map.json` file?

Reply:
No. The downstate NYISO zones are not exact county polygons, but by either splitting a county or combining the two LI counties manually.

Reply:
So this file is self-created then, and not created by the census?

Reply:
Correct. The census county map is used as the input.

Reply:
NYISO provide a list of towns and village names for each subregion.  We could use it to build the subregion, but we have to find the township and village maps.

Reply:
If we made this ourselves, then I don't think it makes sense to point to the census website as the source of this file. We can list it as self-created and describe how we made it (e.g. which counties were combined, which were split, etc.).

Reply:
The description is updated and pushed.